k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -229,7 +234,7 @@ function deleteCortxPVs()
                 kubectl delete pv ${pvc_line[0]}
             fi
         fi
-    done <<< "$(kubectl get pv -A)"
```

Comment:
Use `<<<` with a string will cause the while loop to be entered even if the string is empty. This causes errors or misleading messages if the inner loop is not testing for empty strings. Using command substitution `< <()` instead does not have this problem.